example_scripts/example_runScripts.py

```diff
@@ -43,9 +44,25 @@
 from fitting_benchmarking import do_fitting_benchmark as fitBenchmarking
 from results_output import save_results_tables as printTables
 
-
 # SPECIFY THE SOFTWARE/PACKAGE CONTAINING THE MINIMIZERS YOU WANT TO BENCHMARK
-software = 'mantid'
+software = 'scipy'
```

Comment:
@wathen thanks for this first go. Thinking about it more I would suggested changes which we can discuss at 3pm today:

* remove from software_options 'minimizer_list' and rename 'json_file' to 'minimizer_list_file'
* rename minimzers.json to minimers_list_default.json
* To discussed today: should we insist on example_scripts to load minimizers_list info from file?

Alternative is to rename minimizers.json to fitbenchmarking_config.json, where the later becomes a placeholder for all configurations which can be overwritten